bruno/collections/Rafiki/Rafiki Admin APIs/Approve Incoming Payment.bru

```diff
@@ -0,0 +1,34 @@
+meta {
+  name: Approve Incoming Payment
+  type: graphql
+  seq: 47
+}
+
+post {
+  url: {{RafikiGraphqlHost}}/graphql
+  body: graphql
+  auth: none
+}
+
+body:graphql {
+  mutation ApproveIncomingPayment($input: ApproveIncomingPaymentInput!) {
+  	approveIncomingPayment(input:$input) {
+      id
+      approvedAt
```

Comment:
```suggestion
      payment
```
since it doesnt look like were exposing `approvedAt` in the schema

Reply:
Oh yes, this was just my check of that in the schema.